trusted-committer-article.md

```diff
@@ -0,0 +1,232 @@
+      they are more time constrained compared to doing Open Source after office
+      hours.
+- It is therefore super important to make the process for making contributions
+  and for onboarding contributors as frictionless as possible to avoid wasting
+  a contributors time. 
+- There are a number of things a TC can do to achieve that state
+    - have a good readme.md, providing detailed instructions on how to get,
+      build, test and use the SW developed by the InnerSource community.
+    - have a good contribute.md which outlines what is expected of the
+      contributor to make a contribution. It should answer common questions
+      like
+        - Who do I contact in case of questions and how to reach her?
+        - How do I submit a bug report or feature request?
+        - What is the definition of done for a contribution; code, tests, docs?
+        - What are conventions for code style, branching or committ messages, e. g.?
+        - What is expected of me in terms of supporting contributed code after
+          the contribution was accepted?
+        - What are the process steps that govern contributions?
+        - Is there a code of conduct or other guidelines to how the community
+          operates?
+- As a TC, I have to make sure there is someone in the community who is
+  available to answer questions of users and potential contributors.
+- Actively publishing a list of "Newbie Tasks" to actively solicit contributions,
+  rather than waiting for them to happen.
+- Also make it clear that non-code contributions are welcome, such as writing
+  documentation, organizing events or creating artwork.
+- Gig-Marketplace as a good example of how to do this.
+
+## Uplevelling Community Members
+
+- identifying talent
+- encouraging growth
+- mentoring community members (e. g. during PRs)
+
+## Advocating The Communities Needs
+
+- One major difference of InnerSource compared to Open Source software
+  development is the degree of freedom communities have. 
+- Naturally, an InnerSource community is more constrained because it has to fit
+  in the corporate context.
+- There are varying degrees of freedom, depending on whether your InnerSource
+  initiative focuses more on permissionless innovation or on shared development
+  of reusable components.
+- In any case, there can be times when the company interests are at odds with
+  those of the community. And while companies are more concerned with the
+  bottom line and thus the products produced by an InnerSource community,
+  InnerSource communities, on the other hand understand that a healthy
+  community is a precondition for healthy code. This is why the Apache Software
+  Foundation, after which many InnerSource initiative were modeled, has the
+  motto "Community over Code".
+- It is this potential area of conflict, where the TC plays a vital role.
+- One example I remember is when I developed a system for remotely controlling
```

Comment:
I see your point, @rrrutledge . IMO this scenario is more relevant to the TCs role, though, since it's not about features, but rather about the SW health, which falls into the purview of the TC, rather than the PO. What do you think?

Reply:
I think that makes sense - wasn't clear to me that `iron out a couple of wrinkles` were related to SW health and not features.